core/artwork.go

```diff
@@ -127,6 +128,12 @@ func (a *artwork) getArtwork(ctx context.Context, id string, path string, size i
 		if err != nil {
 			log.Warn(ctx, "Error extracting image", "path", path, "size", size, err)
 			reader, err = resources.FS.Open(consts.PlaceholderAlbumArt)
+
+			if size != 0 {
```

Comment:
You should also check for `err == nil` here

Reply:
You mean like so?
```go
if (size != 0 && err == nil)
```

Reply:
Yes. if it returns error, you should not call resizeImage with the streamer (it may even be `nil`)